diff --git a/t_rax/version.py b/t_rax/version.py
index ee362e1..4ee4729 100644
--- a/t_rax/version.py
+++ b/t_rax/version.py
@@ -15,11 +15,12 @@ def get_version():
         try:
             with open(os.path.join(dir_path, '__version__'), 'r') as fp:
                 version = fp.readline()
-        except ImportError:
-            version = "0.5.0"
+        except Exception:
+            version = "1.4.2"
     else:
+        pass
         # trying to freeze the current version into a python file which gets loaded in case it is not accessible
-        with open(os.path.join(dir_path, '__version__'), 'w+') as fp:
-            fp.write(version)
+        #with open(os.path.join(dir_path, '__version__'), 'w+') as fp:
+            #fp.write(version)
 
-    return version
\ No newline at end of file
+    return version
